### PR TITLE
Clean up dependencies

### PR DESCRIPTION
**Pull request by tolbrino**

### CODE REVIEW

**Comment by tolbrino** [comment]
@ThomasArts Can we merge this?

File: rebar.config [hunk 1]
```diff
@@ -1,4 +1,4 @@
 {erl_opts, [debug_info]}.
 
-{deps, [ yamerl, jsx,
```
Comment (lucafavatella) [inline rebar.config]: jsx is explicitly used in at least swagger_generate so direct dependency seems more readable.

File: rebar.config [hunk 2]
```diff
@@ -1,4 +1,4 @@
...
-         {jesse, {git, "https://github.com/for-GET/jesse.git", {tag, "1.5.0-rc2"}}} ]}.
+{deps, [ yamerl,
+        {jesse, {git, "https://github.com/for-GET/jesse.git", {tag, "1.5.2"}}} ]}.
```
Comment (lucafavatella) [inline rebar.config]: This bump helps when trying to use newer OTP.

**Comment by tolbrino** [comment]
@lucafavatella This has been tested since the change is in use already in aeternity's master branch.

**Comment by tolbrino** [comment]
@ThomasArts Could you review this change too?

**Pull request merged**